electron-app/src/server/websites/bluesky/bluesky.service.ts

```diff
@@ -261,6 +266,12 @@ export class Bluesky extends Website {
       const postId = postResult.uri.slice(postResult.uri.lastIndexOf('/') + 1);
 
       let friendlyUrl = `https://${server}/profile/${handle}/post/${postId}`;
+
+      // After the post has been made, check to see if we need to set a ThreadGate; these are the options to control who can reply to your post, and need additional calls
+      if (data.options.threadgate != "") {
```

Comment:
nit: != -> !== unless you have an explicit need for the coercion to happen

Reply:
Fixed